backend/tests/prisonerQuickLook.test.js

```diff
@@ -67,7 +68,16 @@ describe('prisoner profile quick look', () => {
 
     offenderSearchApi.getPrisonersDetails = jest.fn().mockResolvedValue([])
 
-    controller = prisonerQuickLook({ prisonerProfileService, prisonApi, telemetry, offenderSearchApi, logError })
+    systemOauthClient.getClientCredentialsTokens = jest.fn().mockReturnValue({})
```

Comment:
Need to test that the systemOauthClient gets called

Reply:
Thanks Jamie, agreed added tests for this now